### PR TITLE
Allow building android output on windows

### PR DESCRIPTION
**Pull request by andrewleech**
As well as (existing) linux/mac/*nix support.
Simply checks the current runtime platform and runs gradlew.bat on windows, gradlew (sh script) on others.

### CODE REVIEW

File: briefcase/android.py [hunk 1]
```diff
@@ -107,11 +108,18 @@ def post_install(self):
...
+                proc = subprocess.Popen([
+                    'cmd.exe', '/c', 'gradlew.bat', 'build'
                 ],
-                cwd=os.path.abspath(self.dir)
-            )
+                    cwd=os.path.abspath(self.dir)
+                )
+            else:
+                proc = subprocess.Popen([
+                        '/usr/bin/env', 'sh', './gradlew', 'build'
```
Comment (eliasdorneles) [inline briefcase/android.py]: Is the `/usr/bin/env sh` bit really needed?

**Comment by eliasdorneles** [comment]
@phildini I believe those instructions could be updated instructing people to run `python setup.py android --start` and `python setup.py android --build` instead of `cd android; ./gradlew run`

**Comment by eliasdorneles** [comment]
Sorry for getting in the way -- looks like we were looking into the PR at the same time :)

File: briefcase/android.py (hunk 1 above)
Comment (andrewleech) [reply briefcase/android.py]: I felt it was safer than assuming the shell script had executable permissions, and also matched the windows cmdline better by declaring the shell it's running in.

**Comment by andrewleech** [comment]
Yep I agree the instructions could definitely do with a cleanup, I'll do that when I get a chance at home.

**Pull request merged**